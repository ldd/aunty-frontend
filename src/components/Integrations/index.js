import React from "react";

const Integrations = () => (
  // simulate is-info color for this body
  <div className="hero-body" style={{ backgroundColor: "#209cee" }}>
    <div className="container">
      <div className="columns is-vcentered is-multiline is-centered">
        <div className="column is-full is-centered has-text-centered">
          <div className="button is-large is-primary">Install Slack App</div>
        </div>
        <div className="column is-half">
          <h1 className="title">Create tasks in seconds</h1>
          <h2 className="subtitle">Simple is best</h2>
        </div>
        <div className="column is-half">
          {/* <img src="/screenshots/integrations-slack-create-task.png" alt="Logo" /> */}
        </div>
        <div className="column is-half">
          {/* <img src="/screenshots/integrations-slack-confirm-task.png" alt="Logo" /> */}
        </div>
        <div className="column is-half">
          <h1 className="title">Check in daily</h1>
          <h2 className="subtitle">Or whenever you need to</h2>
        </div>
      </div>
    </div>
  </div>
);

export default Integrations;
