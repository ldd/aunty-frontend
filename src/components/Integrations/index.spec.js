import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";
import Integrations from ".";

describe("Heading", () => {
  it("renders snapshot [empty]", () => {
    const component = renderer.create(<Integrations />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
