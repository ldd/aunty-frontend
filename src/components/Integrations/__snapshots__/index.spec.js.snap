// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading renders snapshot [empty] 1`] = `
<div
  className="hero-body"
  style={
    Object {
      "backgroundColor": "#209cee",
    }
  }
>
  <div
    className="container"
  >
    <div
      className="columns is-vcentered is-multiline is-centered"
    >
      <div
        className="column is-full is-centered has-text-centered"
      >
        <div
          className="button is-large is-success is-outlined"
        >
          Install Slack App
        </div>
      </div>
      <div
        className="column is-half"
      >
        <h1
          className="title"
        >
          Create tasks in seconds
        </h1>
        <h2
          className="subtitle"
        >
          Simple is best
        </h2>
      </div>
      <div
        className="column is-half"
      >
        <img
          alt="Logo"
          src="/screenshots/integrations-slack-create-task.png"
        />
      </div>
      <div
        className="column is-half"
        style={
          Object {
            "padding": "3.5rem",
          }
        }
      >
        <img
          alt="Logo"
          src="/screenshots/integrations-slack-confirm-task.png"
        />
      </div>
      <div
        className="column is-half"
      >
        <h1
          className="title"
        >
          Check in daily
        </h1>
        <h2
          className="subtitle"
        >
          Or whenever you need to
        </h2>
      </div>
    </div>
  </div>
</div>
`;
